src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj

```diff
@@ -20,11 +20,12 @@
         <RepositoryType>git</RepositoryType>
         <PackageIcon>Blazor-API-500px.png</PackageIcon>
         <GeneratePackageOnBuild Condition="$(Configuration) == 'RELEASE' AND $(GeneratePackage) != 'false'">true</GeneratePackageOnBuild>
-        <TargetFrameworks>net8.0;net7.0;net6.0</TargetFrameworks>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
should we also target .net 9?

Reply:
I'm specifically leaving that off for now, because of the outstanding ticket I have with MS about compilation speeds. More as a reminder to do it when it's really ready. But also, .NET 9 can consume .NET 8 libraries, and they would still get the perf benefits of running on .NET 9, so there's no actual advantage except to grab the latest nuget packages.